themes/custom/droopler_theme/scss/layout/_modifiers.scss

```diff
@@ -142,23 +142,63 @@ $two-columned-header-line-margin: 0.625rem !default;
   flex-direction: column;
 
   &.stripe-sidebar {
+    @include media-breakpoint-up(lg) {
+      .item-1 & {
+        margin-right: 140px;
+      }
+      .item-2 & {
+        margin-left: 140px;
+      }
+    }
+
     .with-price-stripe {
       background-color: $stripe-background-color;
       min-width: 170px;
       height: 100%;
       padding: 0;
       position: absolute;
-      right: 0;
       top: 0;
+
+      .item-1 & {
+        right: 0;
+        @include media-breakpoint-down(md) {
+          right: 0;
+          .with-price-wrapper {
+            justify-content: flex-end;
+            padding-right: 7vw;
+          }
+        }
+        @include media-breakpoint-down(sm) {
+          padding-right: $grid-gutter-width / 2;
+        }
+      }
+      .item-2 & {
+        left: 0;
+        @include media-breakpoint-down(md) {
+          left: 0;
+          .with-price-wrapper {
+            justify-content: flex-start;
+            padding-left: 7vw;
+          }
+        }
+        @include media-breakpoint-down(sm) {
+          padding-left: $grid-gutter-width / 2;
+        }
+      }
+
       @include media-breakpoint-down(md) {
         height: 70px;
-        width: 100%;
+        width: 50vw;
```

Comment:
czy to nie wyłazi za okienko jak jest scrollbar ?

Reply:
Nie, to jest 50vw po uprzednio odjętym scrollbarze

Reply:
Nie ruszałbym już tego, zdaje się że to podejście w tym miejscu nie psuje scrolla.
Sprawdzimy to jeszcze potem